pkg/nuclide-file-tree/lib/FileTreeContextMenu.js

```diff
@@ -13,7 +13,7 @@ import {CompositeDisposable} from 'atom';
 import {EVENT_HANDLER_SELECTOR} from './FileTreeConstants';
 import FileTreeStore from './FileTreeStore';
 
-import {isFullyQualifiedLocalPath} from './FileTreeHelpers';
+import {isAbsolute} from 'path';
```

Comment:
The pattern in Nuclide is to import the path module as 'path' and call functions on it.

```
import path from 'path';

...

path.isAbsolute(node.nodePath) &&
```